gathering-service/src/main/resources/application.yml

```
@@ -19,6 +19,22 @@ spring:
       hibernate:
         show_sql: true
         format_sql: true
+  data:
+    redis:
+      stream-key: stream:gathering_join
+      group-name : gathering_group
```

YAMLlint (1.35.1)

[error] too many spaces before colon

(colons)